lib/ZcapClient.js

```diff
@@ -28,48 +41,77 @@ export class ZcapClient {
    * @param {string} options.baseUrl - The base URL for the client to use
    *   when building invocation request URLs.
    * @param {object} [options.didDocument] - A DID Document that contains
-   *   the `capabilityInvocation` and `capabilityDelegation` verification
-   *   relationships. `didDocument` and `keyPairs`, or `invocationSigner` and
-   *   `delegationSigner` must be provided.
+   *   `capabilityInvocation` and `capabilityDelegation` verification
+   *   relationships; `didDocument` and `keyPairs`, or `invocationSigner` and
+   *   `delegationSigner` must be provided in order to invoke or delegate
+   *   zcaps, respectively.
    * @param {Map} [options.keyPairs] - A map of key pairs associated with
-   *   `didDocument` indexed by key pair. `didDocument` and `keyPairs`, or
-   *   `invocationSigner` and `delegationSigner` must be provided.
+   *   `didDocument` indexed by key pair; `didDocument` and `keyPairs`, or
+   *   `invocationSigner` and `delegationSigner` must be provided in order to
+   *    invoke or delegate zcaps, respectively.
    * @param {object} [options.defaultHeaders] - The optional default HTTP
    *   headers to include in every invocation request.
    * @param {HttpsAgent} [options.agent] - An optional HttpsAgent to use to
    *   when performing HTTPS requests.
    * @param {object} [options.invocationSigner] - An object with a
    *   `.sign()` function and `id` and `controller` properties that will be
-   *   used for signing requests. `invocationSigner` and `delegationSigner`, or
-   *   `didDocument` and `keyPairs` must be provided.
+   *   used for signing requests; `invocationSigner` or `didDocument` and
+   *   `keyPairs` must be provided to invoke zcaps.
    * @param {object} [options.delegationSigner] - An object with a
    *   `.sign()` function and `id` and `controller` properties that will be
-   *   used for signing requests. `invocationSigner` and `delegationSigner`, or
-   *   `didDocument` and `keyPairs` must be provided.
+   *   used for delegating zcaps; `delegationSigner` or `didDocument` and
+   *   `keyPairs` must be provided to delegate zcaps.
+   * @param {LinkedDataSignatureSuiteClass} options.SuiteClass - The LD
+   *   signature suite class to use to sign requests and delegations.
+   * @param {Function} [options.documentLoader] - Optional document loader
+   *   to load suite-related contexts. If none is provided, one will be
+   *   auto-generated if the suite class expresses its required context.
    *
    * @returns {ZcapClient} - The new ZcapClient instance.
    */
   constructor({
     baseUrl, defaultHeaders = {}, agent, didDocument, keyPairs,
-    invocationSigner, delegationSigner
+    invocationSigner, delegationSigner, SuiteClass, documentLoader
```

Comment:
Are options supposed to be in alpha order? or mandatory/optional alpha?

Reply:
mandatory/optional alpha -- and I'll reorder this.